esmvalcore/cmor/fix.py

```diff
@@ -152,8 +148,7 @@ def fix_data(cube,
              short_name,
              project,
              dataset,
-             cmor_table=None,
-             mip=None,
+             mip='',
```

Comment:
what's the reason for changing to an empty string?

Reply:
Good point, I fixed this.